### PR TITLE
Remove labels that don't have any open/closed issues labeled with it

### LINKED ISSUE #933: Clean up unused labels in this repository
I see there are 44 labels currently. Only some of them are used in tickets / PRs.

Note: when https://github.com/cncf/mentoring/labels?page=1&sort=name-asc is checked, label `2022` for example seems unused. That view only shows the number of **open** issues/PRs. We shouldn't delete `2022` for historical reasons.

I think this is some manual work.

### PR DESCRIPTION
**Pull request by aliok**
Remove labels that don't have any* open/closed issues labeled with it.
Some labels like `hold` are like the same but I am keeping it.

Fixes #933

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -68,8 +68,6 @@ labels:
   - name: time-sensitive
     description: issues or PRs that have a due date
     color: b60205
-  - name: getting started
-    color: 3B8924
```
Comment (aliok) [inline .github/settings.yml:72]: Not used.
We have good-first-issue and help-wanted that could replace this.

Comment (nate-double-u) [reply .github/settings.yml:72]: I think this is used in the discussions board (see the Q&A section), but I'm not sure it's well used or helpful.

Comment (nate-double-u) [reply .github/settings.yml:72]: Maybe there is a better label for these kinds of questions/discussions?

Comment (aliok) [reply .github/settings.yml:72]: Hm, never thought about the discussions. Thanks for pointing that out.

I will get "getting started" back in the labels list.

We can decide later what label can be more suitable and work it in a separate PR..

Comment (aliok) [reply .github/settings.yml]: Done

**Review by nate-double-u — APPROVED** [review]
Thanks for the update @aliok!

**Pull request merged**